src/geometry/utility/geometry.cc

```diff
@@ -374,6 +374,17 @@ ClosestPointResult GetClosestPoint(const LineString3d& line_string, const malipu
   return result;
 }
 
+double ComputeDistance(const LineString3d& lhs, const LineString3d& rhs) {
+  const LineString3d& base_ls = rhs.size() > lhs.size() ? rhs : lhs;
+  const LineString3d& other_ls = rhs.size() > lhs.size() ? lhs : rhs;
+  double sum_distances{};
+  for (const auto& base_point : base_ls) {
+    const auto closest_point_res = GetClosestPoint(other_ls, base_point);
+    sum_distances += closest_point_res.distance;
+  }
```

Comment:
Can you replace it with std::accumulate?

https://en.cppreference.com/w/cpp/algorithm/accumulate

Reply:
Indeed. Thanks.

Done.